### PR TITLE
More variance to step_walker

### PR DESCRIPTION
**Pull request by mjmadsen**
Switched walk to walk_max and walk_min
Removed unnecessary parameters in function calls
Added variance if target location is within range (do not land directly on pokestop)

### CODE REVIEW

**Comment by douglascamata** [comment]
@mjmadsen the speed variance needs to be configurable.

**Comment by mjmadsen** [comment]
@douglascamata Changes applied.

**Comment by DiegoRBaquero** [comment]
The speed should be +/- not just -. You can achieve this with `random()*2-1` (-0.999...0.999) instead of just `random()` (0...0.999)

File: pokemongo_bot/step_walker.py [hunk 1]
```diff
@@ -38,7 +39,7 @@ def __init__(self, bot, speed, dest_lat, dest_lng):
 
     def step(self):
         if (self.dLat == 0 and self.dLng == 0) or self.dist < self.speed:
-            self.api.set_position(self.destLat, self.destLng, 0)
+            self.api.set_position(self.destLat + random_lat_long_delta(), self.destLng + random_lat_long_delta(), 0)
```
Comment (DiegoRBaquero) [inline pokemongo_bot/step_walker.py]: Where do these come from?

Comment (mjmadsen) [inline pokemongo_bot/step_walker.py]: utils.py

I figured it would be better if we didn't move to the exact stop coord.

**Comment by mjmadsen** [comment]
@DiegoRBaquero We can switch to +/- variance. My thought is walk is the max speed and we sometimes go slower than that. We could also switch to having a walk_max, walk_min and fetch a speed somewhere in there.

**Comment by DiegoRBaquero** [comment]
I'd prefer a min-max setting rather than a variance. I'm searching for a good distribution to use.

**Comment by mjmadsen** [comment]
@DiegoRBaquero Give me a bit and we should have a working setup for walk_min and walk_max in place of walk. I wasted too much time only to realize that distance_unit is only for display purposes and has no effect on the walk unit.

**Comment by mjmadsen** [comment]
I am new to git and having merge troubles. I'm going to shut this down and reopen after corrections.

**Pull request closed without merging**